data.sql

```diff
@@ -178,7 +178,6 @@ WHERE animals_id = 11;
 COMMIT;
 SELECT * FROM visits;
 
-
 -- Add an email column to your owners table
 ALTER TABLE owners ADD COLUMN email VARCHAR(120);
 
```

Comment:
- [ ] You are required to  create a table between `medical_histories` and `treatments`  (name can be different):
- [ ] Make sure all tables have a PRIMARY KEY set
- [ ] All FOREIGN KEYS have indexes created

Reply:
@DeliceLydia  Sorry about the mistake. My bad, wrong assingment.